### ISSUE TITLE
Documentation freqently contradicts itself regarding remembered options

### ISSUE THREAD

**Issue by Ajedi32** [rubygems/bundler-site#389]
From https://bundler.io/v1.16/man/bundle-config.1.html#REMEMBERING-OPTIONS:

> Flags passed to bundle install or the Bundler runtime, such as --path foo or --without production, are not remembered between commands. 

https://bundler.io/v1.16/man/bundle-config.1.html#CONFIGURATION-KEYS

> For instance, passing the --without flag to bundle install(1) prevents Bundler from installing certain groups specified in the Gemfile(5). [...] Additionally, subsequent calls to bundle install(1) remember this setting and skip those groups.

https://bundler.io/v1.16/man/bundle-install.1.html#OPTIONS:

> `--standalone[=<list>]`
>
> [...]
>
> Using this option implicitly sets path, which is a [remembered option][REMEMBERED OPTIONS].

> `--with=<list>`
>
> [...] If a group is given that is in the remembered list of groups given to --without, it is removed from that list.

> `--without=<list>`
>
> [...] If a group is given that is in the remembered list of groups given to --with, it is removed from that list.

There may be more, but those are a few examples I observed.

From the behavior I've personally observed in Bundler, remembered options do indeed exist and both `--deployment` and `--with` are among them.

**Comment by segiddins** [comment]
Yeah, options are remembered in Bundler 1, but in Bundler 2 they will not be -- this might be explain why they're inconsistent -- we might have gotten ahead of ourselves ;)

**Comment by rubymorillo** [comment]
@segiddins do we think there's enough of an inconsistency that we should call it out in the docs? I wonder where we think we should add a note or something to make it clear to readers that options _are_ remembered in Bundler 1.xx?

**Comment by segiddins** [comment]
The problem is that the docs were writing are about Bundler 2, my guess is we accidentally bavkported something we shouldn’t have

**Comment by rubymorillo** [comment]
@segiddins should we close this out? If we need to remove anything from v1.16, though, lmk!

**Comment by Ajedi32** [comment]
The contradictions still exist in the v1.16 docs, so unless that documentation is no longer supported I wouldn't recommend closing this issue.

**Comment by rubymorillo** [comment]
Sounds good @Ajedi32! We're in the process of finding a fix. Thanks for the quick response

**Comment by deivid-rodriguez** [comment]
This was fixed by https://github.com/rubygems/bundler/pull/7242. Closing!
